<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnagramSolver</class>
 <widget class="QWidget" name="AnagramSolver">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anagram Solver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/icon16.png</normaloff>:/img/icon16.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPlainTextEdit" name="inputWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Enter a word here</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="enterAnagramLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>164</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Enter Anagram : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>inputWindow</cstring>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outputWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Solved anagrams appear here.</string>
   </property>
  </widget>
  <widget class="QLabel" name="solutionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>164</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#55aa00;&quot;&gt;Solution(s) :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>outputWindow</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="solveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Solve</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeTakenLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:12pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Elapsed time (s) :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>elapsedTimeWindow</cstring>
   </property>
  </widget>
  <widget class="QTextBrowser" name="elapsedTimeWindow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Time taken to find solution</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Clear contents of all fields.</string>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>360</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>About Anagram Solver</string>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputWindow</tabstop>
  <tabstop>solveButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>outputWindow</tabstop>
  <tabstop>elapsedTimeWindow</tabstop>
  <tabstop>aboutButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>outputWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>AnagramSolver</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>elapsedTimeWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>inputWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
